base-image/entrypoint.py

```diff
@@ -56,10 +58,7 @@ def load_metadata(metadata_path: str) -> dict:
             won't contain metadata but will contain an error message under an 'error' key.
         """
         try:
-            if os.path.splitext(metadata_path)[1] in ('.yml', '.yaml'):
```

Comment:
This is the correct code. It looks like you didn't update your local copy before branching and are reverting to an older version